chemprop/train/molecule_fingerprint.py

```diff
@@ -36,6 +36,12 @@ def molecule_fingerprint(args: FingerprintArgs, smiles: List[List[str]] = None)
 
     #set explicit H option and reaction option
     reset_featurization_parameters()
+    if args.atom_descriptors == 'feature':
+        set_extra_atom_fdim(train_args.atom_features_size)
+
+    if args.bond_features_path is not None:
+        set_extra_bond_fdim(train_args.bond_features_size)
```

Comment:
@MariBerry sorry, I set you up for a problem here. You also need to import `set_extra_bond_fdim`